ansible/roles/eos/tasks/ceos.yml

```diff
@@ -6,13 +6,13 @@
 - name: set force_restart=yes for ceos container
   set_fact: force_restart=yes
 
-- name: set farce_restart=no for ceos container
+- name: set force_restart=no for ceos container
   set_fact: force_restart=no
   when: snmp_data.ansible_facts.ansible_sysname is defined
 
 - include_tasks: ceos_config.yml
 
-- name: Create cEOS container ceos_{{ vm_set_name }}_{{ inventory_hostname }}
+- name: Create cEOS container
```

Comment:
Will this change result in less debug info in ansible log?

Reply:
It's for reduceing helpless and misleading info.
We're not using a explicit loop for all the VMs, all containers created in one task(this one),
so it will be "Create cEOS container ceos_vms24-3_VM2400" but all contains creation info inside it.